frontend/views/containers/proposals/ProposalTemplate.vue

```diff
@@ -46,7 +46,7 @@
           :class='submitStyleNonProposal'
           v-if='!shouldPropose'
           @click='submit'
-          :disabled='disabled'
+          :disabled='disabled || !isGroupCreator'
```

Comment:
It's not clear to me why this condition is being added. Anyone should be able to create a proposal.

Reply:
```
button-submit(
  key='change'
  :class='submitStyleNonProposal'
  v-if='!shouldPropose'
  @click='submit'
  :disabled='disabled || !isGroupCreator'
  data-test='submitBtn'
) {{ submitTextNonProposal }}
```

As the button title says, it's submit button without making proposal. This button is visible when there are less than 3 members in the group. So if he is not group creator, he shouldn't make group settings just like `mincome`, `distributionDate`, `votingRule`, ...
`Add Member`, and `Remove Member` have different user flow so we don't need to take care of them.

Reply:
> This button is visible when there are less than 3 members in the group

Where is this check done? `:disabled` is being set to `true` regardless of the value of `disabled` parameter, because of the `||`. So it will always be disabled if we are not the group creator.

Reply:
@taoeffect 
This `v-if` line in the above does the check there. 
```js
button-submit(
  ...
  v-if='!shouldPropose'
  ...
)
```